docs/blog/2021-08-31-seeding-databases-tutorial.md

```diff
@@ -0,0 +1,733 @@
+playing with databases and data models, it is something that is often hacky. In
+fact there are very few tools that let you iterate quickly on data models, much
+less *developer-friendly* tools.
+
+The core issue at hand is that between iterations on ideas and features, we will
+need to make small and quick changes to our data model. What happens to our
+databases and the data in them in that case? Migration is sometimes an option
+but is notoriously hard and may not work at all if our changes are significant.
+
+For development purposes the quickest solution is seeding our new data model
+with mock data. That way we can test our changes quickly and idiomatically.
+
+## Generate data for your data model
+
+At [Synth][getsynth] we are building a declarative test data generator. It lets
+you write your data model in plain **zero-code** [JSON][json] and seed many
+relational and non-relational databases with mock data. It is
+completely [free and open-source][synth-repo].
+
+Let's take our [blog's data model](#prisma-is-awesome) and seed a
+development [Postgres][postgres] database instance with `synth`. Then we can
+make our development cycle very short by using an [npm script](#npm-scripts)
+that sets it all up for us whenever we need it.
+
+### Installing `synth`
+
+We'll need the `synth` command-line tool to get started. From a terminal, run:
+
+```bash
+$ curl -sSL https://getsynth.com/install | sh
+```
+
+This will run you through an install script for the [binary release][binary]
+of `synth`. If you prefer installing from source, we got you: head on over to
+the [Installation][installation]
+pages of the official documentation.
+
+Once the installer script is done, try running
+
+```bash
+$ synth --version
+synth 0.5.3
+```
+
+to make sure everything works. If it doesn't work, add `$HOME/.local/bin`
+to your `$PATH` environment variable with
+
+```bash
+$ export PATH=$HOME/.local/bin:$PATH
+```
+
+and try again.
+
+### Synth schema
+
+Just like [Prisma][prisma] and its schema DSL, `synth` lets you write down your
+data model with zero code.
+
+There is one main difference: the `synth` schema is aimed at the generation of
+data. This means it lets you specify the semantics of your data model in
+addition to its entities and relations. The `synth` schema has an understanding
+of what an email, a username, an address are; whereas the Prisma schema only
+cares about top-level types (strings, integers, etc).
+
+Let's navigate to the top of our [example project's][repo-complete] directory
+and create a new directory called `synth/` for storing our `synth` schema files.
+
+```
+├── package.json
+├── package-lock.json
+├── tsconfig.json
+├── prisma/
+├── synth/
+└── src/
+``` 
+
+Each file we will put in the `synth/` directory that ends in `.json` will be
+opened by `synth`, parsed and interpreted as part of our data model. The
+structure of these files is simple: each one represents a [table][table] in our
```

Comment:
If you're going to change this up to MongoDB as you mention, you should change the wording from `table` to `collection`.